Documentation/op-guide/container.md

```diff
@@ -21,10 +21,10 @@ sudo rkt trust --prefix coreos.com/etcd
 # gpg key fingerprint is: 18AD 5014 C99E F7E3 BA5F  6CE9 50BD D3E0 FC8A 365E
 ```
 
-Run the `v3.0.6` version of etcd or specify another release version.
+Run the `v3.1.12` version of etcd or specify another release version.
```

Comment:
should these just use `latest` so they don't go out of date?

Run the `latest` version of etcd or specify a specific release (e.g., `v3.1.12`).
?